Post/PostController.swift

```diff
@@ -10,90 +10,111 @@ import Foundation
 
 class PostController {
     
-    static let shared = PostController()
+    static let baseURL = URL(string: "https://dm-post.firebaseio.com/posts/")
```

Comment:
I think it's fine to use a `!` here so `baseURL` is not optional. We know that `URL(string: "https://dm-post.firebaseio.com/posts/")` should never fail.

Reply:
Ok that makes sense. I can update that.